docs/concepts/how-celestia-works/data-availability-layer.md

```diff
@@ -59,10 +59,10 @@ feasible for resource-limited light nodes. However, in order to validate
 block headers, Celestia light nodes need to download the 4k intermediate
 Merkle roots.
 
-For a block data size of n bytes, this means that every light node must
-download O(n) bytes. Therefore, any improvement in the bandwidth capacity
-of Celestia light nodes has a quadratic effect on the throughput of Celestia's
-DA layer.
+For a block data size of n bytes, this means that every light node must download
+O(sqrt(n)) bytes. Therefore, any improvement in the bandwidth capacity of
```

Comment:
[for reviewers] from the previous paragraph

> However, in order to validate block headers, Celestia light nodes need to download the 4k intermediate Merkle roots.

Given the original data square is `n` bytes and the original data square is `k * k` (so `k = sqrt(n)`), the extended data square is `2k * 2k`, and the intermediate merkle roots are `4k`. Downloading `4k` of data is equal to downloading `4 * sqrt(n)` data. Since `4` is a constant term, the big O is `sqrt(n)`